source/multiple-mongosyncs.txt

```diff
@@ -290,7 +270,10 @@ acts as the source cluster:
 
 - If you have not already done so, issue the :ref:`commit
   <c2c-api-commit>` command to each ``mongosync`` instance and wait
-  until all of the commits to finish.
+  until all of the commits to finish. To check if the sync process has been 
+  committed, issue the :ref:`progress <c2c-api-progress>` command to any 
+  ``mongosync`` instance and see if the ``state`` field has the value 
```

Comment:
actually @ajhuh-mdb , could we change this to instead say that the user should issue the progress command to all mongosync instances and check the state field for every progress response? I'm not super confident if the user can just check any instance's progress response